README.md

```diff
@@ -11,7 +11,10 @@
 ```py
 import neatnet
 
-simplified = neatnet.neatify(gdf)
+# Milton Keynes, UK
+original = neatnet.utils.fetch_milton_keynes()
+
+simplified = neatnet.neatify(original)
```

Comment:
I hate being a pain, but I am not sure I like this. It is compely unclear without running the code what is `original`. While `gdf` indicates a geodataframe. Yeah, it runs now but not sure it sends the message as it should.